Makefile

```diff
@@ -1,15 +1,38 @@
 .PHONY: all check-style unittest generate build clean build-image operator-sdk yaml
 
+# Current Operator version - used for bundle
+VERSION ?= 1.6.0
```

Comment:
I believe this version is now `1.8.0`. Sorry, I am sure it was correct at the time you started working on the PR.